ion-schema/src/isl/version_based_isl/constraints/all_of.rs

```diff
@@ -0,0 +1,16 @@
+use crate::isl::version_based_isl::{IslTypeRef, IslVersionTrait};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct AllOf<V: IslVersionTrait> {
+    type_refs: Vec<IslTypeRef<V>>,
```

Comment:
suggestion—this should be a Bag data type, but I'm fine with punting on it for now.